File: scripts/algorithms/F/Find Numbers with Even Number of Digits/Find Numbers with Even Number of Digits.java
class Solution 
{
    public int findNumbers(int[] nums) 
    {
        int count = 0;
        for(int val : nums)
        {
            if((val>9 && val<100)  ||  (val>999 && val<10000)  || val==100000  )
                count++;
        }
        return count;
    }
}

File: scripts/algorithms/F/Find Numbers with Even Number of Digits/Find Numbers with Even Number of Digits.cpp
// Runtime: 11 ms (Top 48.50%) | Memory: 9.9 MB (Top 12.31%)
class Solution {
public:
    int findNumbers(vector<int>& nums) {
        int count = 0;

        for(auto it:nums)
        {
            int amount = 0;
            while(it>0)
            {
                amount++;
                it /= 10;
            }
            if (amount % 2 == 0)
            {
                count++;
            }
        }
        return count;
    }
};